lib/avtp_pipeline/openavb_common/avb.c

```diff
@@ -67,30 +63,27 @@ int pci_connect(device_t * igb_dev)
 		snprintf(devpath, IGB_BIND_NAMESZ, "%04x:%02x:%02x.%d",
 				dev->domain, dev->bus, dev->dev, dev->func);
 		err = igb_probe(igb_dev);
-		if (err)
+		if (err) {
 			continue;
-
+		}
 		printf("attaching to %s\n", devpath);
 		err = igb_attach(devpath, igb_dev);
 		if (err) {
-			printf("attach failed! (%s)\n", strerror(err));
+			printf("attach failed! (%s)\n", strerror(err)); /*errno or err ??*/
```

Comment:
The comment should probably be removed, and I added comments mentioning where errno should be changed to err.